tests/unittest/_torch/thop/parallel/test_cute_dsl_moe.py

```
@@ -154,6 +224,414 @@ def test_moe_sort(num_tokens: int, top_k: int, ep_size: int, tile_size: int):
+        top_k=1,
+        local_expert_offset=slot_start,
+        local_num_experts=num_local_experts,
+        tile_tokens_dim=tile_size,
+    )
+    direct_metadata = torch.ops.trtllm.moe_metadata_from_expert_counts(
+        expert_counts=counts,
+        capacity=capacity,
+        tile_size=tile_size,
+    )
+
+    for old_tensor, direct_tensor in zip(old_metadata, direct_metadata):
```

Ruff (0.16.1)

[warning] `zip()` without an explicit `strict=` parameter

Add explicit value for parameter `strict=`

(B905)